lib/modules/manager/asdf/extract.spec.ts

```diff
@@ -330,6 +331,14 @@ dummy 1.2.3
             depName: 'php',
             extractVersion: '^php-(?<version>\\S+)',
           },
+          {
+            currentValue: '7.26.2',
+            datasource: 'npm',
+            packageName: 'pnpm',
+            versioning: 'semver',
+            depName: 'pnpm',
+            extractVersion: '^v(?<version>\\S+)',
```

Comment:
```suggestion
```

I don't think this is necessary now we switched to the npm datasource (and it might actually break it)